docs/content/using/sample_stateful_container.md

```diff
@@ -2,24 +2,20 @@
 
 You can use the CSI (Container Storage Interface) driver for running stateful containers with a storage volume provisioned from IBM® block storage systems.
 
-These instructions illustrate the general flow for a basic configuration required for running a stateful container with volumes provisioned on storage system.
+These instructions illustrate the general flow for a basic configuration required for running a stateful container with volumes provisioned on a storage system.
 
 **Note:** The secret names given are user specified. To implement order and help any debugging that may be required, provide system type indicators to each secret name when managing different system storage types.
 
 Use this information to run a stateful container on StatefulSet volumes using either file systems or raw block volumes.
 
-1. Create an array secret, as described in [Creating a Secret](../configuration/csi_ug_config_create_secret.md).
+1. Create an array secret, as described in [Creating a Secret](../configuration/creating_secret.md).
 
-2. Create a storage class, as described in [Creating a StorageClass](../configuration/csi_ug_config_create_storageclasses.md).
+2. Create a storage class, as described in [Creating a StorageClass](../configuration/creating_volumestorageclass.md).
 
-    **Remember:** The `SpaceEfficiency` values for Spectrum Virtualize Family are: `thick`, `thin`, `compressed`, or `deduplicated`. These values are not case specific.
-    
-    For DS8000 Family systems, the default value is `none`, but can be set to `thin`, if required. These values are not case specific. For more information, see [Creating a StorageClass](../configuration/csi_ug_config_create_storageclasses.md).
-    
-    This parameter is not applicable for IBM FlashSystem A9000 and A9000R systems. These systems always include deduplication and compression.
+     **Note:** Use the `SpaceEfficiency` parameters available for your storage system, as specified in the [`SpaceEfficiency` parameter definitions per storage system type]
```

Comment:
are these square brackets meant for some formatting? they look strange to me

Reply:
it was unfinished work that needed testing (on a separate branch) 
it's updated now

done

Reply:
nice!